Gruntfile.js

```diff
@@ -130,10 +130,6 @@ module.exports = function(grunt) {
      },
 
     shell: {
-      'build-require': {
-        command: './require.sh'
-      },
-
```

Comment:
Looks like it ran a script that was deleted about a year ago when the repo dropped requirejs--see https://github.com/18F/regulations-site/commit/3ad9df710583f5d694aa3f0118ca41dd00fcba61. There aren't any references to this grunt task left in the project (which is good!), so deleting should be safe.